packages/performance/src/definitions.ts

```diff
@@ -105,3 +148,144 @@ export interface IsEnabledResult {
    */
   enabled: boolean;
 }
+
+export interface PutAttributeOptions {
```

Comment:
```suggestion
/**
 * @since 6.3.0
 */
export interface PutAttributeOptions {
```

Please also add the `@since` tag also to interfaces.